src/components/JUPSettingsMenu/JUPSettingsMenu.tsx

```diff
@@ -91,10 +114,55 @@ const JUPSettingsMenu: React.FC = () => {
           </MenuItem>
         </SLink>
       </Menu>
+
+      <JUPDialog title="About" isOpen={isOpenAboutMenu} closeFn={handleCloseAboutDialog}>
+        <Box sx={{ height: "400px", width: "600px" }}>
+          <StyledCard variant="outlined">
+            <Stack sx={{ margin: "10px" }} direction="row">
+              <CardMedia sx={{ objectFit: "scale-down" }} component={"img"} height="80" image="../assets/logo512.png"></CardMedia>
```

Comment:
i'm surprised at these assets uri's working. i suppose it makes sense since assets are automatically bundled. perhaps these should be referenced from the public folder as a root instead?

Reply:
after side discussion agreed to keep these as-is since they're unlikely to change and they're small enough to be considered acceptable to include in the bundle